client/src/pages/CreateSurvey/Questions/Question.jsx

```diff
@@ -92,16 +138,34 @@ const Question = ({ questionIndex }) => {
             <TextField
               fullWidth
               required
-              error={Boolean(title && title.length > 280)}
+              error={Boolean(
+                questionTitleIsBiggerThanMaxValue ||
+                  questionTitleIsSmallerThanMinValue ||
+                  questionTitleIsEmpty,
+              )}
               helperText={
-                title &&
-                title.length > 280 &&
-                'Question must be less than 280 characters!'
+                (questionTitleIsBiggerThanMaxValue &&
+                  `Question title must be less than 280 characters!`) ||
+                (questionTitleIsSmallerThanMinValue &&
+                  `Question title must be more than 2 characters!`) ||
+                (questionTitleIsEmpty && `Question title is required.`)
               }
               value={title || ''}
               name='title'
               label='Question'
-              onChange={setQuestionData}
+              onChange={
+                (e) =>
+                  setQuestionData(
+                    e,
+                    'text',
+                    'title',
+                    title,
+                    2,
+                    280,
+                    `question${questionIndex}`,
+                  )
```

Comment:
This inline function is quite hard too read easily, maybe making it a const would help?